### PR TITLE
add msu +/- on trading message

### PR DESCRIPTION
**Pull request by blag001**
image

### CODE REVIEW

File: src/ctxcontent/services/analyzer/TradeMessagesAnalyzer.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    this.#messages = messageCallable();
+
+    this.#parseTradeMessages();
+  }
+
+  #getTradesMessages() {
+    const messages = [];
+    this.#messages.forEach((message) => {
+      if (
+        parseInt(message.querySelector(".rawMessageData")?.getAttribute("data-raw-messagetype")) !==
```
Comment (baiiko) [inline src/ctxcontent/services/analyzer/TradeMessagesAnalyzer.js:31]: Too many querySelector, one time only

File: src/ctxcontent/services/analyzer/TradeMessagesAnalyzer.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+          cargo.crystal * (isIncomingRessources ? 1 : -1),
+          cargo.deuterium * (isIncomingRessources ? 1 : -1),
+        ],
+        sourceplayerid: parseInt(message.querySelector(".rawMessageData")?.getAttribute("data-raw-sourceplayerid")),
+        targetplayerid: parseInt(message.querySelector(".rawMessageData")?.getAttribute("data-raw-targetplayerid")),
+      };
+
+      addStandardUnit(trades[msgId], message);
+
+      /*OGIData.trades = trades;*/
```
Comment (baiiko) [inline src/ctxcontent/services/analyzer/TradeMessagesAnalyzer.js:103]: Missing to uncomment ?

**Review by baiiko — COMMENTED** [review]
Plz fix too many querySelector is important

File: src/ctxcontent/services/analyzer/TradeMessagesAnalyzer.js (hunk 2 above)
Comment (blag001) [reply src/ctxcontent/services/analyzer/TradeMessagesAnalyzer.js:103]: This will be step 2, with a new Stat UI like "Fight" 
For now I don't persist

**Review by Bishop341-B — APPROVED** [review]

**Pull request merged**